### ISSUE TITLE
SIGINT/Ctrl-C while profiling should gracefully finish

### ISSUE THREAD

**Issue by mmarchini** [mmarchini-oss/node-observe#13]
SIGINT/Ctrl-C while profiling should gracefully finish the profiler and write the results to stdout/file. Two SIGINT/Ctrl-C should stop the program.

**Comment by RafaelGSS** [comment]
Hey! Just to clarify, is it about close the debugger listener after a command right?

**Comment by mmarchini** [comment]
Yes, it would stop the profiler, capture the data and send to stdout/file (depending on flags). `perf` has a similar behavior.

**Comment by RafaelGSS** [comment]
I take a quick look today on this but seems not supported by nodejs as well. Just local close could be done, maybe sent another signal and provide in docs a default handler to client use? (do not sound good for me).

Maybe can be a great feature on nodejs, what do you think?

Ref: https://github.com/nodejs/help/issues/835
